packages/react-core/src/components/Content/examples/Content.md

```diff
@@ -62,3 +62,11 @@ HTML elements wrapped by `<Content>` are styled by the content component.
 ```ts file="./ContentVisited.tsx"
 
 ```
+
+### Editorial content
+
+Editorial styling increases the font size of body text and small text by one tier, increasing body text to large and small text to the previous body text size.
```

Comment:
```suggestion
Editorial styling increases the font size of body text and small text by 1 tier, where body text becomes "lg" and small text becomes "md". To applying editorial styling, use `isEditorial`.
```
Is it fair to just say small text becomes md, since we mention that body becomes lg in this example? It seems clearer, but not sure what others think

Also, would it make sense to use the sizing shorthand here?